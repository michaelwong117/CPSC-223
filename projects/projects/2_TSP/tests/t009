#!/bin/bash
# File not found

trap "/usr/bin/killall -q -u $USER ./TSP 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./TSP ]; then
  echo './TSP is missing or not executable'
  echo './TSP is missing or not executable' 1>&2
  exit 1
fi


/c/cs474/bin/run -stdout=/dev/null -stderr=stderr.out ./TSP do_not_submit_a_file_with_this_name.in < /dev/null
cat stderr.out
