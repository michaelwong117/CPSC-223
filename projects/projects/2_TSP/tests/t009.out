TSP: could not open do_not_submit_a_file_with_this_name.in
